python/pyspark/sql/streaming.py

```diff
@@ -1464,11 +1491,82 @@ def start(self, path=None, format=None, outputMode=None, partitionBy=None, query
         else:
             return self._sq(self._jwrite.start(path))
 
+    def toTable(self, tableName, format=None, outputMode=None, partitionBy=None, queryName=None,
+                **options):
+        """
+        Streams the contents of the :class:`DataFrame` to the output table.
+
+        A new table will be created if the table not exists. The returned [[StreamingQuery]]
+        object can be used to interact with the stream.
```

Comment:
This one as well. I think it's fine to match the docs with Scaladoc. 

btw nit `[[StreamingQuery]]` -> `` :class:`StreamingQuery` ``